integration-tests/smoke/ccip/ccip_gas_price_updates_test.go

```diff
@@ -102,28 +88,110 @@ func Test_CCIPGasPriceUpdates(t *testing.T) {
 		t.Logf("chainFee1 (stored in chain2): %v", chain1FeeNow)
 		t.Logf("chainFee2 (stored in chain1): %v", chain2FeeNow)
 
-		if !priceDeviationChecked {
-			// make sure there was a price update for chain2 when price deviation was reached
-			if chain2FeeNow.Value.Cmp(initialChain2Fee.Value) == 0 {
-				t.Logf("chainFee2 not updated: %v original=%v", chain2FeeNow, initialChain2Fee.Value)
-				return false
-			}
-			require.NotEqual(t, chain2FeeNow.Timestamp, initialChain2Fee.Timestamp)
-			priceDeviationChecked = true
+		chain1Changed := chain1FeeNow.Value.Cmp(initialChain1Fee.Value) != 0 &&
+			chain1FeeNow.Timestamp > initialChain1Fee.Timestamp
+		chain2Changed := chain2FeeNow.Value.Cmp(initialChain2Fee.Value) != 0 &&
+			chain2FeeNow.Timestamp > initialChain2Fee.Timestamp
+		if chain1Changed {
+			t.Logf("chainFee1 changed: %v prev:%v", chain1FeeNow, initialChain1Fee.Value)
 		}
+		if chain2Changed {
+			t.Logf("chainFee2 changed: %v prev:%v", chain2FeeNow, initialChain2Fee.Value)
+		}
+
+		if chain1Changed && chain2Changed {
+			t.Logf("Calculated fees updated on both chains because expiration is reached")
+			return true
+		}
+		return false
+	}, tests.WaitTimeout(t), gasPriceExpiry)
+}
+
+// price reaches some deviation threshold
+func Test_CCIPGasPriceUpdatesDeviation(t *testing.T) {
+	ctx := testhelpers.Context(t)
+	callOpts := &bind.CallOpts{Context: ctx}
+
+	// Big expiry time to make sure that the update was due to price deviation
+	var gasPriceExpiry = 30 * time.Minute
+	e, _, _ := testsetups.NewIntegrationEnvironment(t,
+		testhelpers.WithOCRConfigOverride(func(params v1_6.CCIPOCRParams) v1_6.CCIPOCRParams {
+			if params.CommitOffChainConfig != nil {
+				params.CommitOffChainConfig.RemoteGasPriceBatchWriteFrequency = *config.MustNewDuration(gasPriceExpiry)
+			}
+			return params
+		}),
+	)
+	state, err := changeset.LoadOnchainState(e.Env)
+	require.NoError(t, err)
+	testhelpers.AddLanesForAll(t, &e, state)
+
+	allChainSelectors := maps.Keys(e.Env.Chains)
+	assert.GreaterOrEqual(t, len(allChainSelectors), 2, "test requires at least 2 chains")
```

Comment:
```suggestion
	require.GreaterOrEqual(t, len(allChainSelectors), 2, "test requires at least 2 chains")
```

Require here so that the test fails right away on this invalid precondition.